repository/meeting-general.js

```diff
@@ -73,19 +68,13 @@ const reopenMeeting = async (meetingURI) => {
   const query = `
   PREFIX besluit: <http://data.vlaanderen.be/ns/besluit#>
   PREFIX besluitvorming: <http://data.vlaanderen.be/ns/besluitvorming#>
-  PREFIX mulit: <http://mu.semte.ch/vocabularies/typed-literals/>
-  PREFIX ext:  <http://mu.semte.ch/vocabularies/ext/>
 
   DELETE {
-    ${sparqlEscapeUri(meetingURI)} besluitvorming:behandelt ?oldAgenda ;
-      ext:finaleZittingVersie ?oldStatus .
-  }
-  INSERT {
-    ${sparqlEscapeUri(meetingURI)} ext:finaleZittingVersie "false"^^mulit:boolean .
+    ${sparqlEscapeUri(meetingURI)} besluitvorming:behandelt ?oldAgenda 
   }
+
   WHERE {
     ${sparqlEscapeUri(meetingURI)} a besluit:Vergaderactiviteit .
-    OPTIONAL { ${sparqlEscapeUri(meetingURI)} ext:finaleZittingVersie ?oldStatus . }
     OPTIONAL { ${sparqlEscapeUri(meetingURI)} besluitvorming:behandelt ?oldAgenda . }
```

Comment:
`OPTIONAL` no longer makes sense here, since `<meeting> besluitvorming:behandelt ?oldAgenda` is the only triple we want to remove in this query.